packages/console/src/pages/ConnectorDetails/index.module.scss

```diff
@@ -0,0 +1,51 @@
+@use '@/scss/underscore' as _;
+
+.container {
+  > *:not(:first-child) {
+    margin-top: _.unit(4);
+  }
+}
+
+.container .header {
+  padding: _.unit(8);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  > *:not(:first-child) {
+    margin-left: _.unit(6);
+  }
+
+  .logo {
+    width: _.unit(19);
+    height: _.unit(19);
+    border-radius: _.unit(4);
+  }
+
+  .metadata {
+    flex: 1;
+
+    > div {
+      display: flex;
+      align-items: center;
+
+      &:not(:first-child) {
+        margin-top: _.unit(2);
+      }
+
+      > *:not(:first-child) {
+        margin-left: _.unit(2);
+      }
```

Comment:
```
div + div {
  margin-left: _.unit(2);
  margin-top: _.unit(2);
}

Reply:
this lines are copied from @gao-sun , I am gonna leave it for now because I am not sure which is better.